### PR TITLE
feat: ollama integration 

### PR DESCRIPTION
**Pull request by sachinsachdeva**
Initial Attempt at solving [Ollama Integration](https://github.com/deepset-ai/haystack/issues/6514)

### CODE REVIEW

File: integrations/ollama/src/ollama_haystack/generator.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        json_response = requests.post(
+            url=self.generation_endpoint,
+            json={
+                "prompt": prompt,
+                "model": self.model_name,
+                "stream": False,
+                "options": generation_kwargs,
+            },
+        ).json()
+
```
Comment (AlistairLR112) [inline integrations/ollama/src/ollama_haystack/generator.py:64]: I think a request.raise_for_status should be added here, for example, the model may not exist and a 404 code would be generated

**Comment by sachinsachdeva** [comment]
@AlistairLR112 looking at your PR looks like you are slightly ahead of me. 
Not sure what's the best way to proceed now, I can drop my PR and we can continue with the one you created .

**Comment by AlistairLR112** [comment]
You can help me!

For example, I am struggling to get the tests running because of relative/absolute import issues :(

**Comment by sachinsachdeva** [comment]
Sure I have a fix for that , lets continue this discussion in your PR

**Comment by sachinsachdeva** [comment]
Continuing instead in https://github.com/deepset-ai/haystack-core-integrations/pull/132

**Pull request closed without merging**